src/app/static/src/app/components/plots/table/TableReshapeData.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-    <table-display :plot="plot"
+    <table-display v-if="tableMetadata"
```

Comment:
type of `tableMetadata` could be `undefined` and we dont want to render that anyway so v-if here